packages/views/signoff.py

```diff
@@ -53,6 +55,36 @@ def signoff_package(request, name, repo, arch, revoke=False):
             return render(request, '403.html', status=403)
         signoff, created = Signoff.objects.get_or_create_from_package(
                 package, request.user)
+        signoffs = Signoff.objects.for_package(package).filter(
+                revoked__isnull=True)
+
+        signed_off_by = []
+        for s in signoffs:
+            signed_off_by.append(User.objects.get(id=s.user_id))
```

Comment:
So two things here, getting the `User` object is pointless because `s.user` would give it to you.

But more importantly, this isn't useful since it's only used for the template, so I would prefer if this variable was removed and `signoffs` was passed directly to the template.  Then the template just needs to do `s.user.username` instead.